packages/manager/modules/pci/src/projects/project/training/jobs/info/logs/logs.html

```diff
@@ -1,26 +1,43 @@
 <p
-    data-ng-if="!$ctrl.job.isTerminal()"
-    data-translate="pci_projects_project_training_job_logs_text_1"
+    data-ng-if="$ctrl.currentJob.isPreRunning()"
+    data-translate="pci_projects_project_training_job_logs_text_pre_running"
 ></p>
 
-<p>
-    {{:: 'pci_projects_project_training_job_logs_text_2' | translate }}
+<p
+    data-ng-if="$ctrl.currentJob.isPostRunning()"
+    data-translate="pci_projects_project_training_job_logs_text_3"
+></p>
+
+<div data-ng-if="$ctrl.currentJob.isTerminal()">
+    <p data-translate="pci_projects_project_training_job_logs_text_5"></p>
+    <p>
+        {{:: 'pci_projects_project_training_job_logs_text_4' | translate }}
+    </p>
```

Comment:
Probably not yours here but we noticed a mixed usage between the `translate` directive/filter.

We tend to avoid using following syntax `{{`.

Maybe here it could be turned into:
```suggestion
    <p data-translate="pci_projects_project_training_job_logs_text_4'"></p>
```

Reply:
Fixed